doc/man7/EVP_CIPHER-SM4.pod

```diff
@@ -37,6 +37,12 @@ The following algorithms are available in the default provider:
 This implementation supports the parameters described in
 L<EVP_EncryptInit(3)/PARAMETERS>.
 
+=head1 NOTES
+
+The XTS mode implementation does not support streaming. That means to obtain
+correct results there can be only one L<EVP_EncryptUpdate(3)> or
+L<EVP_DecryptUpdate(3)> call after the initialization of the context.
+
```

Comment:
I am not sure what SM4 XTS code does in terms of the blocksize, but it does have a stream update also,.

Reply:
I'll look.

Reply:
I am assuming it has the CTS problem also with partial blocks.

Reply:
There are 2 paths - one for streaming asm and the other calls CRYPTO_xts128_encrypt() (The asm one is a bit harder to figure out - The other one needs blocksize until the final part).

Reply:
Dooh there is a other path called stream_gb

Reply:
Looks like it is very similar (Documented in man3/EVP_CIPHER_CTX_ctrl.pod See "xts_standard")

Reply:
Yep, it is.